meta-resin-common/recipes-containers/docker-disk/docker-resin-supervisor-disk.bb

```diff
@@ -8,6 +8,7 @@ SUPERVISOR_REPOSITORY_armv7ve = "resin/armv7hf-supervisor"
 SUPERVISOR_REPOSITORY_aarch64 = "resin/aarch64-supervisor"
 SUPERVISOR_REPOSITORY_x86 = "resin/i386-supervisor"
 SUPERVISOR_REPOSITORY_x86-64 = "resin/amd64-supervisor"
+SUPERVISOR_REPOSITORY_intel-quark = "resin/i386-npl-supervisor"
```

Comment:
I think this should be "i386-nlp-supervisor" instead of "i386-npl-supervisor" ; see https://github.com/resin-io/resin-supervisor/blob/master/base-image/build/conf/machine/i386-nlp.conf